registration.php

```diff
@@ -1,11 +1,35 @@
 <?php
-/*
-  @Author : Mfsi_Annapurnaa
-  @purpose : Registration form layout.
-  : Update operaton on the emplolyee data
+
+/**
+ * @Author  : Mfsi_Annapurnaa
+ * @purpose : Registration form layout and Update operaton on the emplolyee data
  */
 
-require_once('config/dbConnect.php');
+require_once('config/queryOperation.php');
+require_once('config/session.php');
+
+$obj = new queryOperation();
+$objSes = new session();
+$objSes->start();
+$result = $objSes->checkSession();
+
+if (!$result)
```

Comment:
No need of this if else condition.
can simply just set default value for $update = FALSE;
and then write else part as if condition with if ($result)